taskcluster_util/taskcluster_download.py

```diff
@@ -60,6 +60,13 @@ def show_latest_artifacts(self, artifact_downloader, task_id):
         for artifact in artifacts_list:
             print('### {}| {}'.format(artifact.get('contentType').ljust(width), artifact.get('name').ljust(width)))
 
+    def _get_output_directory_path(self):
+        abs_dest_dir = os.path.abspath(self.options.dest_dir) if self.options.dest_dir else os.getcwd()
```

Comment:
hi Askeing, do we need to consider the write permission of given path? ..if yes, we could use os.access(path_name ,os.W_OK) to check ..